Gemfile

```diff
@@ -52,4 +52,4 @@ end
 
 gem 'jsonapi-serializer'
 #data persistance gem
-
+gem 'sidekiq'
```

Comment:
I'm glad we are using sidekiq, I was very much looking forward to learning about this background technology.